jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/datepicker/TypedDatePicker.java

```diff
@@ -96,7 +96,8 @@ protected void initComponent() {
                 .setWeekdays(getWeekdayNames(locale))
                 .setWeekdaysShort(getShortWeekdayNames(locale))
                 .setMonthNames(getMonthNames(locale))
-                .setFirstDayOfWeek(getFirstDayOfWeek(locale)));
+                .setFirstDayOfWeek(calendarDayOfWeekToVaadinDayOfWeek(getFirstDayOfWeek(locale)))
```

Comment:
Pay attention that not on TypedDatePicker uses `getFirstDayOfWeek` but `TypedDateTimePicker` as well